lib/AST/Module.cpp

```diff
@@ -186,58 +187,92 @@ SourceLookupCache &SourceFile::getCache() const {
 }
 
 template<typename Range>
-void SourceLookupCache::doPopulateCache(Range decls,
-                                        bool onlyOperators) {
+void SourceLookupCache::addToUnqualifiedLookupCache(Range decls,
+                                                    bool onlyOperators) {
   for (Decl *D : decls) {
-    if (auto *VD = dyn_cast<ValueDecl>(D))
+    if (auto *VD = dyn_cast<ValueDecl>(D)) {
       if (onlyOperators ? VD->isOperator() : VD->hasName()) {
         // Cache the value under both its compound name and its full name.
```

Comment:
Not your change, but why? Why cache it under both names?? More explanation would help me.

Reply:
You can look up an operator as `+` or `+(_:_:)`